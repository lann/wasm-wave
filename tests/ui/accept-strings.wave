string("")
string("WAVEğŸŒŠ")
string("\\\'\"\t\n\r")