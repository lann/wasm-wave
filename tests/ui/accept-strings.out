string(val: "")
string(val: "WAVEğŸŒŠ")
string(val: "\\\'\"\t\n\r")
